tests/conftest.py

```diff
@@ -1,6 +1,15 @@
+import os
 import pytest
 from frameioclient import FrameioClient
 
-@pytest.fixture
-def frameioclient(token):
-  return FrameioClient("aaaabbbbccccddddeeee")
+token = os.getenv('FRAME_IO_TOKEN')
+
+# @pytest.fixture
+# def frameioclient(token):
+#   return FrameioClient("aaaabbbbccccddddeeee")
```

Comment:
is this no longer needed?

Reply:
Nope, but I'll clean the test stuff up in the follow-on PR's.